### DISCUSSION TITLE
Struggling to update/install

### DISCUSSION THREAD

**Discussion by mynevermore** [felddy/foundryvtt-docker#1015] [Q&A]
I've just installed Foundry VTT in my docker environment and it's a new version so I'm working through getting everything updated and moving all my games from my local machine to the NAS. Clicking 'Update' on any of the systems seems to get a time out error, and I'm trying to install the PbtA system, with an error reading 'Error: cannot read properties of null (reading 'id')'

I'm a little flummoxed so I'm hoping someone can help.

**Comment by adayguerra** [comment]
I am having the same issue, how you solved?
